packages/iTwinUI-react/esbuild.mjs

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import esbuild from 'esbuild';
+import fg from 'fast-glob';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const argsList = process.argv.length > 2 ? process.argv.slice(2) : [];
```

Comment:
nit: I think slice returns an empty list when the start index is higher than the last index in the list. So maybe the check can be removed.

```suggestion
const argsList = process.argv.slice(2);
```